src/org/intellij/erlang/stubs/index/ErlangBehaviourModuleIndex.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2012-2015 Sergey Ignatov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.intellij.erlang.stubs.index;
+
+import com.intellij.psi.stubs.StringStubIndexExtension;
+import com.intellij.psi.stubs.StubIndexKey;
+import org.intellij.erlang.psi.ErlangModule;
+import org.jetbrains.annotations.NotNull;
+
+public class ErlangBehaviourModuleIndex extends StringStubIndexExtension<ErlangModule> {
+  public static final String INDEX = "behaviour";
```

Comment:
In previous review I advised not to expose a key which you use in module index. You did expose it anyway.

Not only that, you also use this constant for module insertion handler which breaks completion for quoted behaviour names. Please fix it and test it properly.